crepecake/src/main/java/com/google/cloud/tools/crepecake/blob/Blobs.java

```diff
@@ -41,4 +42,9 @@ public static Blob from(String content, boolean hashing) {
   public static Blob from(BlobWriter writer) {
     return new HashingWriterBlob(writer);
   }
+
+  /** Wraps a {@link Blob} in an {@link HttpContent}. */
+  public static HttpContent toHttpContent(Blob blob) {
```

Comment:
This doesn't appear to be an implementation or specific implementation of Blob strictly speaking. Should we limit Blobs to purely the creation of new Blob objects?

Reply:
Yes, I moved this to the http package instead.